change/@uifabric-example-app-base-2019-08-02-14-16-32-showCode-fix.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "fix for showcode",
```

Comment:
nit: This could really be improved, if for e.g. you use the title of the PR here. Example-app-base is actually being published ;)

Reply:
done :)